HeterogeneousCore/CUDAServices/bin/cudaIsEnabled.cpp

```diff
@@ -0,0 +1,31 @@
+#include <algorithm>
+#include <array>
+#include <cstdlib>
+#include <iostream>
+
+#include <cuda_runtime.h>
+
+int main() {
+  int devices = 0;
+  auto status = cudaGetDeviceCount(&devices);
+  if (status != cudaSuccess) {
+    return EXIT_FAILURE;
+  }
+
+  int minimumMajor = 6;  // min minor is implicitly 0
```

Comment:
We are re-enabling support for SM 3.5 to run some of the integration tests on a Kepler card.
Do we need to update that here as well ?

Reply:
I believe we should (otherwise `SwitchProducerCUDA` would disable CUDA modules on such machines). Would `minimumMajor = 3` be sufficient (still leaving out the check on the minor version)?